scilpy/utils/streamlines.py

```diff
@@ -123,11 +124,9 @@ def perform_streamlines_operation(operation, streamlines, precision=None):
     return streamlines, indices
 
 
-def warp_tractogram(streamlines, transfo, deformation_data, source):
-    """
-    Warp tractogram using a deformation map.
+def warp_streamlines(streamlines, transfo, deformation_data, source):
```

Comment:
Should we change this to use an SFT as input right now?

Reply:
You can then use the transfo inside the SFT

Reply:
I fixed it, but I kept the name since it returns streamlines